i18n/zh/docusaurus-plugin-content-docs/current/setup/install-with-keadm.md

```diff
@@ -15,6 +15,25 @@ KubeEdge 对 Kubernetes 的版本兼容性，更多详细信息您可以参考 [
 - 需要超级用户权限（或 root 权限）才能运行。
 - `keadm beta`功能在 v1.10.0 上线，如果您需要使用相关功能，请使用 v1.10.0 及以上版本的 keadm。
 
+## 安装keadm
+有三种方式下载keadm二进制文件，分别是：
+- 从github下载：[github releases](https://github.com/kubeedge/kubeedge/releases)
+kubeEdge Github 官方提供了三种架构的发布版本： arm64, arm和amd64。 请根据您的平台和所需的版本下载正确的软件包
+    ```shell
+    wget https://github.com/kubeedge/kubeedge/releases/download/v1.17.0/keadm-v1.17.0-linux-amd64.tar.gz
+    tar -zxvf keadm-v1.17.0-linux-amd64.tar.gz
+    cp keadm-1.17.0-linux-amd64/keadm/keadm /usr/local/bin/keadm
+    ```
+	
+- 下载dockerhub KebeEdge的官方发行版本
+  ```shell
+  docker run --rm kubeedge/installation-package:v1.17.0 cat /usr/local/bin/keadm > /usr/local/bin/keadm && chmod +x /usr/local/bin/keadm
+  ```
+  
+- 从源代码编译安装
+请参考：[build from source](./install-with-binary#build-from-source)
```

Comment:
There is no Chinese version of `build-from-source`, should supplement it first

Reply:
I have added Chinese version about build-from-source and appended here.